src/_components/AssetTypes/AssetTypesContainer.jsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+
+import AssetTypesComponent from '../../components/AssetTypes/AssetTypesComponent';
+import { ToastMessage } from '../../_utils/ToastMessage';
+
+import { loadSubCategories } from '../../_actions/subcategory.actions';
+import { createAssetType } from '../../_actions/assetTypes.actions';
+import resetToastMessageContent from '../../_actions/resetToastMessage.actions';
+
+export class AssetTypesContainer extends React.Component {
+  constructor(props) {
```

Comment:
The constructor is unnecessary as no state is initialized with `props`. Declare the state as a class variable, i.e.
```
export class AssetTypesContainer extends React.Component {
  state = {
    assetType: '',
    ...
  };
  ...
}

Reply:
Alright.